scss/components/_activity-card.scss

```diff
@@ -183,3 +184,211 @@
   display: block;
   text-align: center;
 }
+
+/** Designs for big activity card **/
+
+.civicase__activity-card--big {
+  box-shadow: 0 3px 8px 0 rgba(49, 40, 40, 0.15);
```

Comment:
I think I used this `rgba` colour too and it's time to document it. Can we move it to a variable if possible? `$card-box-shadow-color`? (just a suggestion)